sync-todo/v2/client/react-native/README.md

```diff
@@ -1,17 +1,20 @@
 # React Native Template App
 
 ## Prerequisites
+
 - Make sure your system is setup to run a React Native application by following the [setup guide](https://reactnative.dev/docs/environment-setup)
 - Set up an Atlas account and [deploy a cluster](https://www.mongodb.com/docs/atlas/tutorial/deploy-free-tier-cluster/)
 - Set up an application in App Services.
-    -  You can use the [`realm-cli`](https://www.mongodb.com/docs/atlas/app-services/cli/) to set up the backend and download a copy of this template:
-		```
-		realm-cli apps create -n "<App Name>" --template "react-native-ts.todo.flex"
-		```
+  - You can use the [`realm-cli`](https://www.mongodb.com/docs/atlas/app-services/cli/) to set up the backend and download a copy of this template:
+
+
+      ```
+      realm-cli apps create -n "<App Name>" --template "react-native.todo.v2"
```

Comment:
My suggestion about changing this name was with the understanding that we were going to a v2 name for these apps. Since it's temp, we probably shouldn't have changed it. But I don't think this needs to block merging the updates - just something to address when we finalize the name at the "real" launch.